udata/tests/reuse/test_reuse_model.py

```diff
@@ -10,9 +10,15 @@
 from udata.core.user.factories import UserFactory
 
 from .. import TestCase, DBTestMixin
+from udata.frontend.markdown import init_app
 
 
 class ReuseModelTest(TestCase, DBTestMixin):
+    def create_app(self):
```

Comment:
I tested the two branches, I couldn't see any difference in test speed with this code.